robusta_krr/utils/object_like_dict.py

```diff
@@ -0,0 +1,29 @@
+class dict_to_object:
```

Comment:
Can we give this a class name format (`DictToObject` or something)

Can you explain why we need this?

Reply:
We can name it DictToObject also, but I thought it is more like a name for a function
I thought about `ObjectLikeDict` also, but I did not like it for some reason

And we need that becuase kubernetes library returns objects for everything but CRDs. So I did this to be able to reuse the same code with CRDs and objects